SPECS/make/make.spec

```diff
@@ -41,11 +42,12 @@ make %{?_smp_mflags} check
 %{_mandir}/*/*
 
 %changelog
-* Sat May 09 00:21:04 PST 2020 Nick Samson <nisamson@microsoft.com> - 4.2.1-4
+* Mon Oct 19 2020 Andrew Phelps <anphel@microsoft.com> 4.2.1-5
+- Fix check test
+* Sat May 09 2020 Nick Samson <nisamson@microsoft.com>  4.2.1-4
```

Comment:
Thank you for fixing this. Could you also remove the double space before the version number?

Reply:
fixed